instances/UBERONParcellation/anteriorThalamicRadiation.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/anteriorThalamicRadiation",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+  "definition": "",
+  "description": "'' is a . It is part of .",
```

Comment:
Those need to be fixed. Either add content (improve content for description) or set to "null".

Reply:
Edit: This comment is true for all instances in the PR.

Reply:
oh oh... I was so tired yesterday that I completely forgot about this open todo :see_no_evil: I'll take care of it this afternoon.